TestingTravis/TestClass.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  testClass.swift
+//  TestingTravis
+//
+//  Created by Zomato on 14/03/19.
+//  Copyright © 2019 tester. All rights reserved.
+//
+
+import Foundation
+
+class TestClass: NSObject {
+    
+    var forceUnwrapMe: String?
+    
+    override init() {
+        print(forceUnwrapMe!)
+    }
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Nakul97/TestingTravis/blob/42f5da818d42b6fbe59b862aeeb22de4b43eacb6/TestingTravis/TestClass.swift#L18"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TestClass.swift:18</code></td>
    </tr>
  </tbody>
</table>